File: Easy/1935. Maximum Number of Words You Can Type/solution.py
class Solution:
    def canBeTypedWords(self, text: str, brokenLetters: str) -> int:
        words = text.split(' ')
        counter = 0
        for word in words:
            for c in word:
                if c in brokenLetters:
                    counter += 1
                    break

        return len(words) - counter

File: Easy/1935. Maximum Number of Words You Can Type/solution.cs
public class Solution {
    public int CanBeTypedWords(string text, string brokenLetters) {
        int counter = 0;
        
        string[] words = text.Split(" ");
        foreach(string word in words){
            foreach(char c in word){
                if(brokenLetters.Contains(c)){
                    counter += 1;
                    break;
                }
            }
        }

        return words.Length - counter;
    }
}